models/stg_reddit_ads.yml

```diff
@@ -281,7 +281,7 @@ models:
       - name: account_id
         description: '{{ doc("account_id") }}'
       - name: campaign_id
-        description: '{{ doc("campaign_id") }}'
+        description: '{{ doc("campaign_id") }}' 
```

Comment:
Should we also provide a not_null test for this field?

Reply:
updated